bedrock-devnet/devnet/__init__.py

```diff
@@ -217,7 +252,7 @@ def devnet_deploy(paths):
             'go', 'run', 'cmd/main.go', 'genesis', 'l2',
             '--l1-rpc', 'http://localhost:8545',
             '--deploy-config', paths.devnet_config_path,
-            '--l1-deployments', paths.addresses_json_path,
+            '--deployment-dir', paths.deployment_dir,
```

Comment:
This one seems [incorrect](https://github.com/ethereum-optimism/optimism/blob/develop/bedrock-devnet/devnet/__init__.py#L173)